api/src/main/java/org/apache/gravitino/policy/Policy.java

```diff
@@ -33,6 +34,20 @@
 @Evolving
 public interface Policy extends Auditable {
 
+  /** The set of metadata object types that the policy can be applied to. */
+  Set<MetadataObject.Type> SUPPORTS_ALL_OBJECT_TYPES =
+      new HashSet<MetadataObject.Type>() {
+        {
+          add(MetadataObject.Type.METALAKE);
```

Comment:
Is it valid to support policy for metalake? As we don't support tagging the metalake, maybe we should align with the tag.